.github/workflows/integration.yml

```diff
@@ -12,12 +12,9 @@ jobs:
     # We cannot use 'actions/setup-python@v4' because it only sets the Python version for non-root users
     # The root user will still be using the latest available Python version provided by Ubuntu
 
-    # NOTE: Inspec repositories are not providing a package for jammy yet
     - name: Install Chef Inspec
       run: |
-        wget https://packages.chef.io/files/stable/inspec/4.41.20/ubuntu/20.04/inspec_4.41.20-1_amd64.deb
-        sudo dpkg -i inspec_4.41.20-1_amd64.deb
-        rm inspec_4.41.20-1_amd64.deb
+        curl https://omnitruck.chef.io/install.sh | sudo bash -s -- -P inspec -v 5.21.29
```

Comment:
Specific reason why to use this? I know it's their recommended way, but piping `curl` to bash when you can simply download a `deb` file seems... more adventurous than necessary?

Reply:
Just saw that this was the recommended way of installing inspec and the script itself is pretty much doing what we do. But you're right, it might be better to just directly download the deb file. I'll revert but keep the inspect update.